python/packages/nisar/cal/corner_reflector.py

```diff
@@ -400,10 +400,11 @@ def filter_crs_per_az_heading(crs, az_heading, az_atol=np.deg2rad(20.0)):
         TriangularTrihedralCornerReflector.
     az_heading : float
         Desired AZ/heading angle in radians w.r.t. geographic North.
-    az_atol : float, default=20.0 degrees
+    az_atol : float, default=pi/6 (30 degrees)
         Absolute tolerance in radians when comparing AZ of CRs with
-        `az_heading`. The default is around 0.5 * HBPW of an ideal
-        triangular trihedral CR (HPBW ~ 40 deg).
+        `az_heading`. The default is slightly larger than the half of the
+        half-power beam width (BPBW) of an ideal triangular trihedral CR
```

Comment:
What does the "B" in "BPBW" stand for? Did you mean "HPBW"?
```suggestion
        half-power beam width (HPBW) of an ideal triangular trihedral CR
```